.ci/packaging.groovy

```diff
@@ -133,7 +133,7 @@ pipeline {
             }
           }
         }
-        stage('DRA Staging') {
+        stage('DRA Staging (if no main)') {
```

Comment:
```suggestion
        stage('DRA Staging (release branches only)') {
```

I think this would be a bit clearer, but approving regardless. Thanks!

Reply:
There is certain limitation in the stage name, let me commit your changes and see if it works as expected